spec/services/idv/steps/in_person/address_step_spec.rb

```diff
@@ -142,4 +142,23 @@
       end
     end
   end
+
+  describe '#updating_address effects on extra_view_variables.updating_address' do
+    let(:address1) { '123 Fourth St' }
```

Comment:
let's use the constant we have for `address1` in this test, it's: `InPersonHelper::GOOD_ADDRESS1`